mitodl/bootcamp-ecommerce
# Bootcamp run certificate models changes

#### What are the relevant tickets?
#1100 

#### What's this PR do?
Implemented the basic certificate model structure required for bootcamp runs

#### How should this be manually tested?
- Checkout the branch
- Run migrations
- Visit the admin panel to verify the model structure for bootcamp run certificates.
